src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -847,6 +847,10 @@ private IEnumerable<ManifestItemWithContext> ExportManifest(HostService hostServ
                         {
                             Logger.LogDiagnostic($"Processor {hostService.Processor.Name}: Saving...");
                             m.BaseDir = context.BuildOutputFolder;
+                            if (m.PathRewriter != null)
+                            {
+                                m.File = m.PathRewriter(m.File);
+                            }
                             if (m.FileAndType.SourceDir != m.FileAndType.DestinationDir)
```

Comment:
only one can be occurred.

---
In reply to: [78318231](https://github.com/dotnet/docfx/pull/688#discussion_r78318231) [](ancestors = 78318231)